coding-the-legislation/35_periods.md

```diff
@@ -1,25 +1,103 @@
 # Periods
 
+A period can be a month, a year, `n` successive months or `n` successive years.
```

Comment:
@benjello The relation between a period and definition_period is stated here : `The size of the period is constrained by the class attribute definition_period`

What would you like to be added ?